templates/zerver/emails/email.css

```diff
@@ -20,19 +21,20 @@ body {
     -webkit-text-size-adjust: 100%;
 }
 
-pre {
```

Comment:
We only have one <pre> tag, which has the class `.deactivated-user-text`. We also may encounter <pre> tags within the `missed_message` content, but we do not want to apply CSS to those, so I moved this rules to `.deactivated-user-text`.

Reply:
That sounds reasonable.